src/redux/actionTypes/projects.js

```diff
@@ -50,6 +50,11 @@ const PROJECTS_METADATA_UPDATE = `${PROJECTS}/metadataUpdate`;
  */
 const PROJECTS_METADATA_DELETE = `${PROJECTS}/metadataDelete`;
 
+/**
+ * Load the projects
+ */
+const PROJECTS_LOAD = `${PROJECTS}/load`;
```

Comment:
naming it `PROJECTS_LOAD` makes it ambigious as to whether you mean `loaded` or `loading`. I would rename this to `PROJECTS_LOADED` and `${PROJECTS}/loaded` to avoid this situation.